experimental/rest_client.go

```diff
@@ -37,6 +37,7 @@ func NewRestClient(url string, headers map[string]string) Client {
 		client: &http.Client{
 			Transport: &http.Transport{
 				TLSClientConfig: &tls.Config{
+					MinVersion:    tls.VersionTLS12,
```

Comment:
Was this caught by linter? What was the message?

Reply:
@marefr Yes, the gosec linter complains that anything below TLS 1.2 is insecure.

Reply:
I'm not sure. But a bit surprised we haven't got hit by this in Grafana repo, but maybe we have?

Reply:
@marefr Maybe it's one of the disabled gosec rules. I don't remember which one it is. Many gosec rules make sense to disable as part of general linting, since they're too opinionated, but I thought this rule (catching too relaxed TLS support) made sense.

Reply:
Since it's in experimental package, this should be fine